lib/wasix/src/utils/mod.rs

```diff
@@ -1,20 +1,22 @@
+mod dummy_waker;
 mod owned_mutex_guard;
 pub mod store;
 mod thread_parker;
 
-mod dummy_waker;
-pub use self::dummy_waker::WasiDummyWaker;
-
-use std::collections::BTreeSet;
+#[cfg(feature = "js")]
+pub(crate) mod web;
```

Comment:
Is this `js` or `web`?

Should we align the naming?

This code should potentially also work in non-browser environments , like Deno/Node, so `js` seems like a more sensible name.

On the other hand, `web` might then require some different setup over a generic `js`?

I primarily want to prevent us boxing ourselves into a suboptimal naming that will require a breaking change.
Thoughts?

Reply:
It's funny that you mention this... We were originally planning to rename the `js` feature to `web` in the main `wasmer` crate, but backed it out because `web` is just as bad a name as `js` (https://github.com/wasmerio/wasmer/issues/3844).

I'm inclined to leave things as-is for now. The `wasmer-wasix` crate is still very much unstable and the public API needs an overhaul, so we can probably leave it until we are ready to do `1.0`.